### ISSUE TITLE
Invalid type: 'container' must be a String or HTMLElement

### ISSUE THREAD

**Issue by designorant** [ONSdigital/census-atlas#163]
[Another map related error caught here](https://app.netlify.com/sites/ons-data-map/deploys/6218c2de560a6000085cbd58#L331):

```
11:54:04 AM:      Error: The following error originated from your application code, not from Cypress.
11:54:04 AM:   > Invalid type: 'container' must be a String or HTMLElement.
11:54:04 AM: When Cypress detects uncaught errors originating from your application it will automatically fail the current test.
11:54:04 AM: This behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.
11:54:04 AM: https://on.cypress.io/uncaught-exception-from-application
11:54:04 AM:       at new o (https://deploy-preview-162--ons-data-map.netlify.app/_app/chunks/vendor-fc058ce4.js:501:121429)
11:54:04 AM:       at HTMLLinkElement.h.onload (https://deploy-preview-162--ons-data-map.netlify.app/_app/chunks/Map-4ab23e1b.js:10:44796)
```

_Originally posted by @designorant in https://github.com/ONSdigital/census-atlas/issues/162#issuecomment-1050797269_

**Comment by James-Nitron** [comment]
I'll take a look!

**Comment by designorant** [comment]
Fixed in #167
